engine/apps/slack/tests/test_scenario_steps/test_distribute_alerts.py

```diff
@@ -89,15 +89,15 @@ def test_timeout_error(
 def test_alert_shooting_no_channel_filter(
     mock_post_alert_group_to_slack,
     make_slack_team_identity,
+    make_slack_channel,
     make_organization,
     make_alert_receive_channel,
     make_alert_group,
     make_alert,
 ):
     slack_team_identity = make_slack_team_identity()
-    organization = make_organization(
-        slack_team_identity=slack_team_identity, general_log_channel_id="DEFAULT_CHANNEL_ID"
-    )
+    slack_channel = make_slack_channel(slack_team_identity)
```

Comment:
this should fix failing test (channel slack id is checked below (line 111)):
```suggestion
    slack_channel = make_slack_channel(slack_team_identity, slack_id="DEFAULT_CHANNEL_ID")
``` 
🙏